src/server/pfs/pretty/pretty.go

```diff
@@ -158,6 +161,60 @@ func PrintCommitInfo(w io.Writer, commitInfo *pfs.CommitInfo, fullTimestamps boo
 	fmt.Fprintln(w)
 }
 
+// PrintCommitSetInfo pretty-prints jobset info.
+func PrintCommitSetInfo(w io.Writer, commitSetInfo *pfs.CommitSetInfo, fullTimestamps bool) {
+	// Aggregate some data to print from the jobs in the jobset
+	success := 0
+	failure := 0
+	var created *types.Timestamp
+	var modified *types.Timestamp
+	for _, commitInfo := range commitSetInfo.Commits {
+		if commitInfo.Finished != nil {
+			// TODO(2.0 required): when the errored commit PR is merged, this needs to change to use that field
+			if commitInfo.Description == pfs.EmptyStr {
+				failure++
+			} else {
+				success++
+			}
+		}
+
+		if created == nil {
+			created = commitInfo.Started
+			modified = commitInfo.Started
+		} else {
+			if commitInfo.Started.Compare(created) < 0 {
+				created = commitInfo.Started
+			}
+			if commitInfo.Started.Compare(modified) < 0 {
+				modified = commitInfo.Started
+			}
+		}
+	}
+
+	fmt.Fprintf(w, "%s\t", commitSetInfo.CommitSet.ID)
+	fmt.Fprintf(w, "%d\t", len(commitSetInfo.Commits))
+	fmt.Fprintf(w, "%s\t", pretty.ProgressBar(8, success, len(commitSetInfo.Commits)-success-failure, failure))
+	if created != nil {
+		if fullTimestamps {
+			fmt.Fprintf(w, "%s\t", created.String())
+		} else {
+			fmt.Fprintf(w, "%s\t", pretty.Ago(created))
+		}
+	} else {
+		fmt.Fprintf(w, "-\t")
+	}
```

Comment:
I guess this whole `if` would be a useful utility function, although it seems unlikely to change too much